src/dummy-data.json

```diff
@@ -0,0 +1,310 @@
+{
+  "history": [
+    {
+      "stationName": "add new",
```

Comment:
The `AddNew` component I imagined as being a separate component, so no need to put this data here.

Reply:
We thought about that but then we realised it can potentially be the same component, but with a different onClick event?

Reply:
It will be styled differently, and behave in quite a different way, so probably best to separate the two things out.